content/docs/refs-and-the-dom.md

```diff
@@ -194,84 +193,21 @@ function CustomTextInput(props) {
 
 在极少数情况下，你可能希望从父组件访问子节点的 DOM 节点。通常不建议这样做，因为它会破坏组件的封装，但偶尔也可用于触发焦点或测量子 DOM 节点的大小或位置。
 
-虽然你可以[向子组件添加 ref](#adding-a-ref-to-a-class-component),但这不是一个理想的解决方案，因为你只能获取组件实例而不是 DOM 节点。并且，它还在函数式组件上无效。
```

Comment:
Deleted the whole paragraphs because I cannot find the counterpart of them in the official documentation. Maybe they are outdated. Please double check before merge.